app/src/main/java/in/testpress/testpress/authenticator/ResetPasswordVerificationActivity.java

```diff
@@ -0,0 +1,70 @@
+package in.testpress.testpress.authenticator;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.afollestad.materialdialogs.GravityEnum;
+import com.afollestad.materialdialogs.MaterialDialog;
+
+import javax.inject.Inject;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import in.testpress.testpress.Injector;
+import in.testpress.testpress.R;
+import in.testpress.testpress.core.TestpressService;
+import in.testpress.testpress.models.ResetPassword;
+import in.testpress.testpress.util.SafeAsyncTask;
+
+public class ResetPasswordVerificationActivity extends AppCompatActivity {
```

Comment:
Why is this called `ResetPasswordVerificationActivity`?? What is the `verification` here? Should be just `ResetPasswordActivity`